store/tikv/region_cache.go

```diff
@@ -300,15 +301,15 @@ func (c *RegionCache) GetTiKVRPCContext(bo *Backoffer, id RegionVerID, replicaRe
 	}
 
 	regionStore := cachedRegion.getStore()
-	var store *Store
-	var peer *metapb.Peer
-	var storeIdx int
+	var sg RouteStrategy
 	switch replicaRead {
 	case kv.ReplicaReadFollower:
-		store, peer, storeIdx = cachedRegion.FollowerStorePeer(regionStore, followerStoreSeed)
+		sg = cachedRegion.KeepSendLast
 	default:
-		store, peer, storeIdx = cachedRegion.WorkStorePeer(regionStore)
+		sg = cachedRegion.SendCurrRetryNextOnFail
```

Comment:
If it's a TiFlash request and does not set the `ReplicaReadFollower`. Is the region only sent to one store?